src/components/sticky-object.js

```diff
@@ -60,6 +60,8 @@ AFRAME.registerComponent("sticky-object", {
   },
 
   _onGrab() {
+    if (!this.el.components.grabbable || this.el.components.grabbable.data.maxGrabbers === 0) return;
```

Comment:
Was going to point out that this was weird since things that aren't `grabbable` shouldn't get grab events (not actually true) and things with `maxGrabbers` of 0 should also not get grab events (also not true). Fooled by logic!

Reply:
/shrug